### ISSUE TITLE
FileInput looks misaligned

### ISSUE THREAD

**Issue by m4tx** [zostera/django-bootstrap4#40]
FileInput looks misaligned when compared to other form fields: 
![screenshot from 2017-10-29 12-11-01](https://user-images.githubusercontent.com/3128220/32142937-4d952e56-bca2-11e7-8d27-624e66f8883f.png)
(this is a screenshot of horizontal form, but the problem exists in default layout as well)

The problem is the `fix_clearable_file_input` function in `FieldRenderer` class. It looks like this:
```
def fix_clearable_file_input(self, html):
    # TODO This needs improvement
    return '<div class="row bootstrap4-multi-input"><div class="col-xs-12">{html}</div></div>'.format(
        html=html
    )
```
TODO says it needs improvement - certainly, and the funny thing is, the improvement may look like this:
```
def fix_clearable_file_input(self, html):
    return html
```
Yes, the input looks okay with the code like that:
![screenshot from 2017-10-29 12-14-49](https://user-images.githubusercontent.com/3128220/32142971-c79189d4-bca2-11e7-970e-8e2dec1a3ce0.png)
However, I believe that the code above was also supposed to make the field span the entire available width, and that's why I'm using something like that in my code:
```
def fix_clearable_file_input(self, html):
    return html.replace('class="', 'class="w-100 ')
```
However, this is a very ugly solution and this is why I haven't included a PR fixing that. This needs a bit more investigation on how to make this look properly.

**Comment by dyve** [comment]
Anyone that is willing to take a look at this and maybe turn it into a documented PR? If not it will probably have to wait until we run into this problem in a project.

**Comment by dyve** [comment]
Was this fixed by #146 / 106d89716c9fc59af16ac93cf10a148db8e5efd4?

**Comment by m4tx** [comment]
I am not sure if it was fixed by this exact commit, but the problem is gone with django-bootstrap4 1.1.1, so I believe this issue can be closed.
